src/controllers/index.js

```diff
@@ -6,7 +6,14 @@ const login = require('./login');
 const error = require('./error');
 
 const router = express.Router();
+const { verify } = require('./middlewares/jwtVerification');
+const { createCookie } = require('./middlewares/createCookie');
```

Comment:
unnecessary. because it sent on app.js file as a helper and you will only send the title as a string. @AhmedAl-Almi